### PR TITLE
API route for problem category and statements

### PR DESCRIPTION
**Pull request by borsezf2**
API endpoints created

1) fetch list of categories 

2) fetch list of problem statements related with category

### CODE REVIEW

File: backend/urls.py [hunk 1]
```diff
@@ -25,5 +25,6 @@
     re_path(r'^api/v1/hackathon/', include('hackathon.api.urls')),
     re_path(r'^api/v1/docs/', include_docs_urls(title='Isohack API', permission_classes=[], public=True)),
     path('', index, name='index'),
-    re_path(r'^(?:.*)/?$', index)
+    re_path(r'^(?:.*)/?$', index),
```
Comment (shashank-sharma) [inline backend/urls.py:28]: Not needed

File: hackathon/api/views.py [hunk 2]
```diff
@@ -66,3 +68,30 @@ class HackathonEventAPIView(generics.ListAPIView):
...
+    def get_problem_category(request):
+        if request.method == 'GET':
+            category = ProblemCategory.objects.all().values('id','category','description','created_at')
+            # serializer = ProblemCategorySerializer(category,many=True)
+            listed = list(category)
+            return JsonResponse(listed, safe=False)
+        else:
+            print("Only GET is allowed")
+
+    def get_problem_statements(request, category_id):
```
Comment (shashank-sharma) [inline hackathon/api/views.py:85]: Should be separate class API

File: hackathon/api/urls.py [hunk 3]
```diff
@@ -12,4 +12,6 @@
     path('status/', HackathonAppStatus.as_view(), name='hackathon-app-status'),
     path('categories/', ProblemCategoryAPIView.as_view(), name='hackathon-problem-category'),
     path('events/', HackathonEventAPIView.as_view(), name='hackathon-event'),
+    path('problem/category', ProblemCategoryClass.get_problem_category, name='hackathon-problem-categoryAPI'),
```
Comment (shashank-sharma) [inline hackathon/api/urls.py:15]: Should have trailing slash in urls.py

**Review by shashank-sharma — APPROVED** [review]
Overall it completed the work ... Later we will organize if required

File: hackathon/api/views.py (hunk 2 above)
Comment (borsezf2) [reply hackathon/api/views.py:85]: okay, 
i though both had the same parent i.e category hence, will discuss these things before hand next time ,
thanks

**Pull request merged**